src/lib/logs/logs_processor.py

```diff
@@ -67,20 +157,19 @@ def _prepare_context_and_process_message(sfm_queue: Queue, message: ReceivedMess
         return None
 
 
-def _process_message(context: LogsProcessingContext, message: PubsubMessage) -> Optional[LogProcessingJob]:
-    context.self_monitoring.processing_time_start = time.perf_counter()
-    data = message.data.decode("UTF-8")
-    # context.log(f"Data: {data}")
+def _process_message(
+    context: LogsProcessingContext, message: PubsubMessage, ack_id
+) -> Optional[LogProcessingJob]:
+    data = base64.b64decode(message.get("data"))
```

Comment:
Are we sure this message object will always have a "data" field?
If not, the data value will be set to None, which will throw an AttributeError on the next line.